### DISCUSSION TITLE
Standardize Voxel Size in Transform (Train and Inference)

### DISCUSSION THREAD

**Discussion by aksg87** [Project-MONAI/MONAI#3188] [Q&A]
Is there a nice way to standardize voxel sizes for a volume in MONAI? Doing this for 2D is straightforward, but would love to do this for 3D.

For example, we have some input volume `(x, y, z)` where and `z` is the number of slices, and we want to resample to a new volume `(x, y, z')` where `z'` is the new number of slices.

Furthermore, ideally, this transform can be easily reversed during inference.

**Comment by aksg87** [comment]
Perhaps using `scipy.ndimage.zoom` is the best approach for this:

https://stackoverflow.com/questions/47775621/interpolate-resize-3d-array

**Reply by Nic-Ma** [reply]
Hi @aksg87 ,

Thanks for your interest here.
MONAI provides rich transforms to handle medical image, I think you can find some transforms for your case: Zoom, Resize, Spacing, etc. in: https://docs.monai.io/en/latest/transforms.html#zoom
And all of them are inverse transforms, you can refer to this tutorial to invert transforms in inference:
https://github.com/Project-MONAI/tutorials/blob/master/3d_segmentation/torch/unet_inference_dict.py#L70

Thanks.
